app/controllers/users_controller.rb

```diff
@@ -1,3 +1,31 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    if User.find_by id: params[:id] == nil
+      flash[:error] = t "users.error"
+      render :new
+    else
+      @user = User.find_by id: params[:id]
+    end
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t "home.content_1"
```

Comment:
nên đặt tên biến i18n phù hợp với nội dung trả về
ví dụ : 
```suggestion
      flash[:success] = t "home.create_user_successful"
```